src/test/kotlin/com/igorwojda/range/containsrange/challenge.kt

```diff
@@ -4,7 +4,10 @@ import org.amshove.kluent.shouldBeEqualTo
 import org.junit.jupiter.api.Test
 
 private fun containsRange(range1: IntRange, range2: IntRange): Boolean {
-    TODO("not implemented")
+    range2.forEach {
```

Comment:
Is it optimal to check every item in the range? Or maybe just the range start & end? :)

Reply:
aa acum mi-am dat seama ca e range nu o lista de numere random